confectory-core/src/main/java/net/obvj/confectory/Confectory.java

```diff
@@ -0,0 +1,49 @@
+package net.obvj.confectory;
+
+/**
+ * A default {@link ConfigurationContainer}
+ * 
+ * @author @Dautomne (Fernando Tiannamen)
+ * @since 0.1.0
+ */
+public final class Confectory
+{
+
+    /**
+     * The initial {@link ConfigurationContainer} to be loaded as default
+     */
+    protected static ConfigurationContainer DEFAULT_CONFIGURATION_CONTAINER;
```

Comment:
(1) Make it private
(2) Name it with camelCase as this is not a constant. Suggested name: "globalConfigurationContainer"
(3) Also improve the Javadoc with the idea that this is the current object (not exactly the initial)